pipeline/dags/main.py

```diff
@@ -76,6 +76,22 @@ def _geocode():
             )
 
 
+def _export_to_s3(logical_date):
+    import logging
+
+    from airflow.providers.amazon.aws.hooks import s3
+
+    from dag_utils import date
+
+    logger = logging.getLogger(__name__)
+    s3_file_path = f"data/api/{date.local_date_str(logical_date)}.dump"
```

Comment:
1. il faut garder `run_id` (idempotence et pas de collision si + d'une exec par jour)
2. namespacer par `marts` (sur le même niveau d'idée que `raw`)

```suggestion
    s3_file_path = f"data/marts/{date.local_date_str(logical_date)}/{run_id}/api.dump"
```

Reply:
OK !
J'avais prévu de faire des fichiers plats pour simplifier l'accès par simple lecture du timestamp des fichiers (si je veux prendre le plus récent par exemple) mais pourquoi pas suivre la nomenclature actuelle. Ca va juste complexifier un peu le code de récupération de l'autre côté : 
- selectionner le dernier dossier créé ou parser les dates et trier
- pareil dans le sous-dossier mais par run_id
- puis recupérer api.dump

Apres moi ça ne me pose pas de probleme.

Reply:
ouai on devrait pouvoir s'en sortir avec un glob + tri